app/scripts/controllers/channelsmodal.js

```diff
@@ -292,11 +301,21 @@ app.controller('channelBasicInfoCtrl', function ($scope) {
         $scope.urlPattern.regex = false;
       }
     });
+  };
+
+  // if update is true
+  if ($scope.update) {
+    setUrlPattern();
   }else{
     // set default options if new channel
     $scope.channel.type = 'http';
     $scope.channel.authType = 'private';
     $scope.channel.status = 'enabled';
+
+    if ( $scope.channelDuplicate ){
+      setUrlPattern();
```

Comment:
Does this need to be done here? Perhaps we could rather just make the if statement above `if ($scope.update || $scope.channelDuplicate) {`